### DISCUSSION TITLE
Bit of help guys re link from my site

### DISCUSSION THREAD

**Discussion by leedsrhinos** [boxbilling/boxbilling#1222] [General]
hi guys bit of help please 
i have installed boxbilling and think its great i am now trying to configure my website with links to the bb areas 
My problem at moment is this i am trying to configure the domain checker serch here button to go to boxbilling domain page which is 

https://trustedwebhosting.co.uk/account/order/domain-checker
but when i put the link in to the button i get error 
Please take a look and advise or has bb got a domain checker page ?
Thanks

**Comment by leedsrhinos** [comment]
link to page i need help with
https://trustedwebhosting.co.uk/?page_id=268

**Comment by quicklyweb** [comment]
Have you configured individual domain pricing and reseller settings in
boxbilling?

This message contains information intended solely for the addressee, which
is confidential or private in nature. If you are not the intended
recipient, you may not peruse, use, disseminate, distribute or copy this
message or any file attached to this message. Any such unauthorised use, is
prohibited and may be unlawful. If you have received this message in error,
please notify the sender immediately by e-mail or telephone and thereafter
delete the original message from your machine.

**Reply by leedsrhinos** [reply]
Hi yes 
If I copy paste this link https://trustedwebhosting.co.uk/account/order/domain-checker
Direct to browser it works.
But put the link on the check now button on my website it errors.
Have checked and double checked button link and it's same link as above.

**Comment by leedsrhinos** [comment]
another issue what this mean in the blue highlighted box
![Clipboard02](https://user-images.githubusercontent.com/100451638/157116805-fba18518-8ce7-471f-a7ab-11f1abc60ada.jpg)
